src/Upgrade/Infrastructure/VersionControlSystem/Git/Git.php

```diff
@@ -188,15 +196,15 @@ public function createPullRequest(StepsExecutionDto $stepsExecutionDto): StepsEx
             return $stepsExecutionDto;
         }
 
-        $params = [
-            'base' => $this->getBaseBranch(),
-            'head' => $this->getHeadBranch(),
-            'title' => 'Updated to the latest Spryker modules up to ' . date('m/d/Y h:i', time()),
-            'body' => $this->pullRequestDataBuilder->buildBody($composerDiffDto),
-            'auto_merge' => $this->configurationProvider->isPullRequestAutoMergeEnabled(),
-        ];
+        $pullRequestDto = new PullRequestDto(
+            $this->getHeadBranch(),
+            $this->getBaseBranch(),
+            'Updated to the latest Spryker modules up to ' . date('m/d/Y h:i', time()),
+        );
+        $pullRequestDto->setBody($this->pullRequestDataGenerator->buildBody($composerDiffDto));
```

Comment:
Please avoid usage of setters for DTO's as they have to be immutable.

Reply:
++ removed setters